### PR TITLE
Add working Ansible examples

### PR DESCRIPTION
**Pull request by nywilken**
```
cd ./example
packer build .
```

### CODE REVIEW

File: example/docker.json.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+source "docker" "autogenerated_1" {
```
Comment (azr) [inline example/docker.json.pkr.hcl:2]: NITNIT not blocking: we could name this one so that it feels more 'real'

**Review by azr — APPROVED** [review]
Cool !

File: example/docker.json.pkr.hcl (hunk 1 above)
Comment (nywilken) [inline example/docker.json.pkr.hcl:2]:
```suggestion
source "docker" "debian" {
```

File: example/docker.json.pkr.hcl [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  discard = true
+  image   = "debian:jessie"
+}
+
+build {
+  sources = ["source.docker.autogenerated_1"]
```
Comment (nywilken) [inline example/docker.json.pkr.hcl:8]:
```suggestion
  sources = ["source.docker.debian"]
```

**Pull request merged**